deny.toml

```diff
@@ -166,10 +164,13 @@ skip = [
     # prost needs an older version of `petgraph`.
     { name = "petgraph", version = "0.7.1" },
     # chrono-tz
+    { name = "phf", version = "0.12.1" },
```

Comment:
Are we ok with having more entries here? In general I've become hesitant to upgrade libraries that require them.